libgalois/include/katana/DynamicBitset.h

```diff
@@ -43,41 +43,50 @@ namespace katana {
  * Concurrent dynamically allocated bitset
  **/
 class KATANA_EXPORT DynamicBitset {
-  katana::PODResizeableArray<katana::CopyableAtomic<uint64_t>> bitvec;
-  size_t num_bits{0};
+  katana::PODResizeableArray<katana::CopyableAtomic<uint64_t>> bitvec_{};
+  size_t num_bits_{0};
 
 public:
-  static constexpr uint32_t bits_uint64 = sizeof(uint64_t) * CHAR_BIT;
+  static constexpr uint32_t kNumBitsInUint64 = sizeof(uint64_t) * CHAR_BIT;
```

Comment:
`uint32_t` --> `size_t`?

Reply:
As it's a small number, I was thinking about making it a `uint8_t` but didn't. It's a `constexpr`, so I don't think it matters except for readability. If you are strongly in favor of `size_t`, I can make it `size_t`.

Reply:
Yeah, I was just saying for consistency's sake. Don't worry abt it.